File: JWTApi/Models/Question.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace EBET.Models
{
    public class Question
    {
        public int QuestionId { get; set; }
        public string question { get; set; }
        public string Option1 { get; set; }
        public string Option2 { get; set; }
        public string Option3 { get; set; }
        public string Option4 { get; set; }
        public string CorrectAnswer { get; set; }
        public string AnswerDetails { get; set; }
        public int? ImageId { get; set; }
        public int? ImageIdForAnswer { get; set; }
        [ForeignKey("ImageId")]
        public Image Image { get; set; }
        public int? ChapterId { get; set; }
        [ForeignKey("ChapterId")]
        public Chapter Chapter { get; set; }
        public ICollection<QuestionStatus> QuestionStatuses { get; set; }
    }
}

File: JWTAuth/src/app/_models/question.ts
export interface Question{
    questionId: number;
    chapterId: number;
    imageId: number;
    imageUrl: string;
    imageIdForAnswer: number;
    imageUrlForAnswer: string;
    question: string;
    option1: string;
    option2: string;
    option3: string;
    option4: string;
    correctAnswer: string;
    answerDetails: string;
}
